chpl/chpl-resources/src/main/resources-dev/jobs.xml

```diff
@@ -75,13 +94,29 @@
                 </entry>
             </job-data-map>
         </job>
+        
+        <job>
+            <name>summaryStatisticsCreator</name>
+            <group>systemJobs</group>
+            <description>Generates the summary statistics report</description>
+            <job-class>gov.healthit.chpl.scheduler.job.summarystatistics.SummaryStatisticsCreatorJob</job-class>
+            <durability>true</durability>
+            <recover>true</recover>
```

Comment:
Doesn't matter to me - I figured if a job that collected data was not executed due to the a system fault, we should re-try the job.  Otherwise everyone who has a trigger to the associated email job will receive the same data they got previously.  I'll make the change.